components/ui/update-center/MetaData.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2019 SonarSource SA
```

Comment:
Whoops, someone should do something about the license headers 😅 .

Reply:
Good catch. I'll take the opportunity to deal with that in this PR.